bagri-server/bagri-server-hazelcast/src/main/java/com/bagri/server/hazelcast/impl/DocumentManagementImpl.java

```diff
@@ -1065,9 +1066,9 @@ public DocumentAccessor removeDocument(String uri, Properties props) throws Bagr
 
 	private DocumentAccessor removeDocumentInternal(DocumentKey docKey, Document doc, Properties props) throws BagriException {
 		triggerManager.applyTrigger(doc, Order.before, Scope.delete);
-		Object result = xddCache.executeOnKey(docKey, new DocumentRemoveProcessor(txManager.getCurrentTransaction(), props));
+		Object result = xddCache.executeOnKey(docKey, new DocumentRemoveProcessor(props));

```

Comment:
no, you should pass the Tx to the processor here. It'll be not accessible (via TxManagement) after we'll come to EntryProcessor